### DISCUSSION TITLE
Add create_typescript_schema() for client

### DISCUSSION THREAD

**Discussion by jet10000** [vitalik/django-ninja#1304] [Ideas]
Currently, the model can be automatically converted to Python schema by create_schema, and it is expected to be automatically converted to TypeScript schema. This makes it convenient for client.

```
create_typescript_schema()
create_typescript_zod_schema()
create_dart_schema()
```

**Comment by martinsvoboda** [comment]
Very complex task... Rather use specialized tool like https://orval.dev/
